8Knot/assets/main_layout.css

```
@@ -382,6 +372,99 @@ a.sidebar-section-text {
+
+.login-banner-container {
+    position: fixed;
+    top: 70px;
+    right: 20px;
+    z-index: 1000;
+}
+
+.login-banner-alert {
+    background-color: var(--baby-blue-100);
+    border-color: var(--baby-blue-700);
+    border: 1px solid var(--baby-blue-700);
```

Biome (2.1.2)

[error] Unexpected shorthand property border after border-color



(lint/suspicious/noShorthandPropertyOverrides)